src/views/TreasuryDashboard/components/Graph/TreasuryGraph.tsx

```diff
@@ -0,0 +1,534 @@
+      tickStyle={getTickStyle(theme)}
+    />
+  );
+};
+
+/**
+ * Displays the market value chart and assets table together, along with a toggle
+ * to choose between displaying the market value or liquid backing.
+ *
+ * The assets table will update according to the toggle selection.
+ *
+ * @param param0
+ * @returns
+ */
+export const TreasuryAssets = ({ subgraphUrl, count = DEFAULT_RECORDS_COUNT }: GraphProps) => {
+  const isTreasuryAssetActive = (assets: string): boolean => {
+    return selectedTreasuryAssets === assets;
+  };
+
+  // State variable for the selected tab
+  const [selectedTreasuryAssets, setSelectedTreasuryAssets] = useState(QUERY_TREASURY_MARKET_VALUE);
+  const [isLiquidBackingActive, setIsLiquidBackingActive] = useState(false);
+  // Set the selected treasury assets from search parameters
+  const [searchParams] = useSearchParams();
+  useMemo(() => {
+    // Get the record count from the URL query parameters, or use the default
+    const queryTreasuryAssets = searchParams.get(QUERY_TREASURY) || QUERY_TREASURY_MARKET_VALUE;
+    setSelectedTreasuryAssets(queryTreasuryAssets);
+    setIsLiquidBackingActive(queryTreasuryAssets === QUERY_TREASURY_LIQUID_BACKING);
+  }, [searchParams]);
+
+  const getSearchParamsWithUpdatedTreasuryAssets = (assets: string): string => {
+    return updateSearchParams(searchParams, QUERY_TREASURY, assets).toString();
+  };
+
+  // State variable for the selected index in the chart
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  /**
+   * Uses mouse movement events in the market value chart to record the
+   * current index that the user is hovering over. This is then passed to
+   * the assets table in order to have the contents reflect the current
+   * index (date).
+   *
+   * @param nextState
+   * @param event
+   * @returns
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const onMouseMove: CategoricalChartFunc = (nextState, event) => {
+    if (!nextState.activeTooltipIndex) return;
+
+    setSelectedIndex(nextState.activeTooltipIndex);
+  };
+
+  return (
+    <>
+      <Grid container paddingBottom={2}>
+        <Grid item xs={1} md={2} lg={3} xl={4} />
+        <Grid item xs={10} md={8} lg={6} xl={4} textAlign={"center"}>
+          <TabBar
+            disableRouting
+            items={[
+              {
+                label: t`Market Value`,
+                to: `/dashboard?${getSearchParamsWithUpdatedTreasuryAssets(QUERY_TREASURY_MARKET_VALUE)}`,
+                isActive: isTreasuryAssetActive(QUERY_TREASURY_MARKET_VALUE),
+              },
+              {
+                label: t`Liquid Backing`,
+                to: `/dashboard?${getSearchParamsWithUpdatedTreasuryAssets(QUERY_TREASURY_LIQUID_BACKING)}`,
+                isActive: isTreasuryAssetActive(QUERY_TREASURY_LIQUID_BACKING),
+              },
+            ]}
+          />
+        </Grid>
+        <Grid item xs={1} md={2} lg={3} xl={4} />
+      </Grid>
```

Comment:
This will resolve spacing issues:

```suggestion
      <Box display="flex" flexDirection="row" justifyContent="center" pb={2}>
        <TabBar
          disableRouting
          items={[
            {
              label: t`Market Value`,
              to: `/dashboard?${getSearchParamsWithUpdatedTreasuryAssets(QUERY_TREASURY_MARKET_VALUE)}`,
              isActive: isTreasuryAssetActive(QUERY_TREASURY_MARKET_VALUE),
            },
            {
              label: t`Liquid Backing`,
              to: `/dashboard?${getSearchParamsWithUpdatedTreasuryAssets(QUERY_TREASURY_LIQUID_BACKING)}`,
              isActive: isTreasuryAssetActive(QUERY_TREASURY_LIQUID_BACKING),
            },
          ]}
        />
      </Box>
```
![image](https://user-images.githubusercontent.com/95196612/181263281-35fec988-bd8d-4355-b9c9-06c93005900e.png)

Reply:
Thanks!